modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -0,0 +1,331 @@
+= Configuring Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, studio, logger, logs, log, notifications, errors, debug
+
+You can configure what gets logged, where it gets logged, and how by editing a configuration file that sits in your project.
```

Comment:
"that sits in" -> "that resides in" ? Not sure where it resides though. Maybe that will become evident later.